### ISSUE TITLE
Build fails

### ISSUE THREAD

**Issue by RenaKunisaki** [mwbrown/cc2640r2_blinky#1]
```
$ export CC2640_SDK="/home/rena/.wine/drive_c/ti/simplelink_cc2640r2_sdk_4_10_00_10"
$ make
make: *** No rule to make target '_build/ti/devices/cc26x0/startup_files/startup_gcc.o', needed by 'blinky.elf'.  Stop.
make: *** Waiting for unfinished jobs....
CC main.c
CC board_config.c
```

**Comment by mwbrown** [comment]
Hey there,

My first guess (since I haven't touched this project in almost two years) is that the SDK has changed significantly since this was originally written. Until I've got a chance to look at this with the latest SDK version, I would assume that the GCC startup file is either moved or missing. This project will probably also have to be updated with new JTAG information, etc at the same time depending on the current state of XDS110 OpenOCD support vs. something like like a J-Link.

Edit: specifically, the SDK has gone from 1.50.00.58 at the time of writing to the one you're compiling against, i.e 4.10.00.10... that's a pretty big version jump.

Cheers

**Comment by RenaKunisaki** [comment]
Darn. I don't suppose you know of anyone else who's managed to program the CC2640 with OpenOCD? Especially without TI's massive bloat? Coming from the Teensy, things are so simple there...

**Comment by mwbrown** [comment]
As far as bloat is concerned, my example is still pulling in DriverLib files, which in my experience is where the bulk of the bloat is... you might be better suited writing registers directly depending on which peripherals you are using (except for Bluetooth, since that can't really be shrunk).

**Comment by RenaKunisaki** [comment]
True, the library itself is huge, but at least I could avoid CCStudio and SmartRF and all that.
